dotcom-rendering/src/web/components/Footer.tsx

```diff
@@ -202,7 +212,16 @@ const FooterLinks: React.FC<{
 
 	const rrLinks = (
 		<div css={readerRevenueLinks}>
-			<div id="reader-revenue-links-footer" />
+			<Island deferUntil="idle" clientOnly={true}>
```

Comment:
```suggestion
			<Island deferUntil="visible" clientOnly={true}>
```

Does it work if we use visible for the footer?